proxy/coco/src/project.rs

```diff
@@ -13,6 +13,10 @@ pub use create::{Create, Repo};
 pub mod checkout;
 pub use checkout::Checkout;
 
+/// TODO(finto)
```

Comment:
What's the TODO?

Reply:
It's module level doc. It was left to quiten the clippy. Should be addressed before approved.

Reply:
TODOcument